components/SSI/SBTxWallet/index.tsx

```diff
@@ -146,86 +206,250 @@ function Component() {
         }
     }
 
-    const option = [
-        {
-            key: '',
-            name: 'Select action',
-        },
-        {
-            key: 'Ivms101',
-            name: 'Upload Travel Rule',
-        },
-        {
-            key: 'Verifiable_Credential',
-            name: `Mint Issuer's SBT`,
-        },
+    const fetchPause = async () => {
```

Comment:
let's have a general/global function to fetch the smart contract state for several dapps